requirements.txt

```diff
@@ -1,2 +1,4 @@
+git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
@Irio @cuducos I've removed the `-e` but was wondering if you guys had any thoughts on keeping it locked to a version like I did here? I like that it makes things reproductible and makes it clear when we expect people to update the package on their local installations but I noticed you guys didn't do it on rosie 💭